Mathlib/Data/PEquiv.lean

```diff
@@ -135,8 +135,11 @@ theorem symm_refl : (PEquiv.refl α).symm = PEquiv.refl α :=
 theorem symm_symm (f : α ≃. β) : f.symm.symm = f := by cases f; rfl
 #align pequiv.symm_symm PEquiv.symm_symm
 
+theorem symm_bijective : Function.Bijective (symm : α ≃. β → β ≃. α) :=
+  Equiv.bijective ⟨PEquiv.symm, PEquiv.symm, symm_symm, symm_symm⟩
```

Comment:
nit: please write this as the longer 
```suggestion
  Equiv.bijective { toFun := PEquiv.symm, infFun := PEquiv.symm, left_inv := symm_symm, right_inv := symm_symm }
```
as this is more robust to future refactors of field layouts

Reply:
I assume you'd like me to fix this in the existing manifestastions of this lemma (which use this layout?)

Actually it may be better to use `bijective_iff_has_inverse` because that is equivalent and in some sense avoids this strange "create an Equiv just to say that it is bijective" construction. I would make that change everywhere. (Basically the core reason this is true is "symm_symm", "symm_symm (the other way round)", "thus, bijective" and so these are just different ways of spelling that.)

What do you think?

Reply:
Using `bijective_iff_has_inverse` sounds good, that save you one repetition of the word `symm`